src/components/VideoCard.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Video } from "../types";
+import { ArrowRightCircleFillIcon } from "./ArrowRightCircleFillIcon";
+
+const getDurationInString = (seconds: number) => {
+  if (seconds < 60) {
+    return `${seconds} seconds`;
+  }
+  const minutes = Math.floor(seconds / 60);
+  return `${minutes} minute${minutes > 1 ? "s" : ""}`;
+};
+
+const ONE_DAY = 1000 * 60 * 60 * 24;
+
+type VideoCardProps = {
+  video: Video;
+  onClickCta: VoidFunction;
+};
+
+export const VideoCard = ({ video, onClickCta }: VideoCardProps) => {
+  const tag = React.useMemo(() => {
+    const today = new Date();
+    const publishedDate = new Date(video.publishDate);
+    const diffDays = Math.round(
```

Comment:
Personally, I would use Math.floor and compare against 6 days, instead of 7 for consistency